index.html

```diff
@@ -7,6 +7,7 @@
     <title>有限会社ヤタカ物産　建材各種販売</title>
     <link href="/css/reset.css"  rel="stylesheet" />
     <link href="/css/style.css"  rel="stylesheet" />
+    <link href="/css/common.css"  rel="stylesheet" />
```

Comment:
ありがとう！順序としては、リセット → 共通 → ページ固有のスタイル で読ませると好ましいので、下記の記述順に変更お願いします🙏
```suggestion
    <link href="/css/reset.css"  rel="stylesheet" />
    <link href="/css/common.css"  rel="stylesheet" />
    <link href="/css/style.css"  rel="stylesheet" />
```